Pear/Controller/FeedbackViewController.swift

```diff
@@ -225,9 +225,17 @@ class FeedbackViewController: UIViewController {
     }
 
     @objc private func finishFeedback() {
-        print("finish feedback")
+        guard let netId = UserDefaults.standard.string(forKey: Constants.UserDefaults.userNetId) else { return }
         responses.append(detailTextView.text)
         print(responses)
+        NetworkManager.shared.getMatchHistory(netID: netId).observe {response in
```

Comment:
```suggestion
        NetworkManager.shared.getMatchHistory(netID: netId).observe { response in
```

Space before `response in`